new-frontiers-14/frontier-station-14
# Sector-wide alerts, Frontier-specific alert text

## About the PR



Communications consoles now change a sector-wide alert level, which is properly displayed on PDAs and emergency lights on any available ship.  Alert levels now display Frontier-specific text, with reference to the sector, crews, etc.

This could be merged with the NFSD SOP concept of heat (particular heat levels require active station alert levels), but this is left as a TODO.

## Why / Balance


Been a long time coming.

Fixes #1139.

## How to test


1. Set a Red alert level on Frontier Outpost.
2. Warp to Expeditionary Lodge.  Neat, the lights are on.
3. Wander outside, check your PDA.  Wow, the sector's in Red alert.

## Media


This photo shows a ghost in space, checking their PDA during a Violet alert.  The emergency lights inside the Expeditionary Lodge are on and appropriately set to the alert level.
![image](https://github.com/user-attachments/assets/4df2dd3b-2d50-4930-98d6-529aec9f0c6f)

## Requirements



## Breaking changes


**Changelog**


:cl:
- tweak: Alert levels are now sector-wide, with appropriate announcements.
